packages/app/app/components/PlayQueue/QueueBottomBar/index.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './styles.scss';
+import { Icon } from 'semantic-ui-react';
+import _ from 'lodash';
+
+export const QueueBottomBar= ({
+  addToDownloads,
+  items,
+  currentSong
+}) => {
+
+  const handleDownloads = () => {
+    if ((currentSong !== null && currentSong >= 0) && (items !== null && items !== undefined && items.length > 0)) {
+      addToDownloads(_.get(items, currentSong));
+    }
+  };
+
+  const bottomBarOptions = [{
+    link: '#',
+    iconName: 'triangle left',
+    onClickHandler: null
```

Comment:
what function of other two bar options?

Reply:
Can you comment them out and add a `TODO` comment? I don't want to have buttons that do nothing there, since it could be misleading.

Reply:
Done @nukeop